harbour-counter.pro

```diff
@@ -133,3 +133,6 @@ qm.CONFIG += no_check_exist
 INSTALLS += qm
 
 OTHER_FILES += LICENSE
+
+DISTFILES += \
+    translations/harbour-counter-hu.ts
```

Comment:
This isn't necessary but I can fix that myself - my way of integrating translations into the build is slightly weird :)